CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ CHANGELOG
 
 ## HEAD (Unreleased)
 
+## 0.18.15 (2020-01-14)
+
 * Upgrade to go1.13.x
 
 * Due the necessity to perform many async operations during creation, many parts of an
```

Comment:
Minor - but there’s a typo (“not” instead of “not”) in this entry which is fairly important to the description. 

May also want to remove “go1.13” from changelog as that is not user facing.  (Or put below the big actual change)

Reply:
Will do.